test/src/705-evict/main

```diff
@@ -0,0 +1,111 @@
+cvmfs_test_name="Evict chunks"
+cvmfs_test_autofs_on_startup=false
+cvmfs_test_suites="quick"
+
+TEST705_PRIVATE_MOUNT=
+
+source ./src/705-evict/setup_teardown
+
+
+evict_file() {
+  local mntpnt="$1"
+  local filename="$2"
+  local minimum_chunks="$3"
+
+  echo ""
+  echo "*** START Testing evict_file for file: /$filename"
+
+  echo "   Mounting repo..."
+  private_mount $mntpnt || return $?
+
+  echo "   Accessing files"
+  cat $mntpnt/large > /dev/null
+  cat $mntpnt/small > /dev/null
+
+  local lc_oldCache=$(sudo cvmfs_talk -p ${mntpnt}c/$CVMFS_TEST_REPO/cvmfs_io.$CVMFS_TEST_REPO cache list | grep -e $filename | wc -l)
+  # make sure filehas at least $minimum_chunks in cache
+  [ $lc_oldCache -ge $minimum_chunks ] || return 10  
+
+  echo "   Evicting file $filename"
+  sudo cvmfs_talk -p ${mntpnt}c/$CVMFS_TEST_REPO/cvmfs_io.$CVMFS_TEST_REPO evict "/$filename"
+
+  local lc_newCache=$(sudo cvmfs_talk -p ${mntpnt}c/$CVMFS_TEST_REPO/cvmfs_io.$CVMFS_TEST_REPO cache list | grep -e $filename | wc -l)
+  # make sure file is completly evicted from cache
+  [ $lc_newCache -eq 0 ] || return 11 
+
+
+  echo "   ...Success"
+  echo "   Unmount repo"
+  private_unmount
+  echo "*** FINISHED Testing evict_file for file: /$filename"
+  echo ""
+}
+
+evict_and_readd() {
```

Comment:
```suggestion
evict_and_read() {
```

Reply:
i wanted to say re-add .. but i guess evict_and_read could also work :-D